### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by SSonnyboy** [bookman233/TDAT#4]
There seems to be a bug in your code that is preventing it from running (line 155)
The dimensions returned by your label_relaxation function are:
Torke.size ([128, 10]) This obviously cannot be directly used to compute the loss using nn.CrossEntropyLoss()(ori_output, relaxtion_labell.float ())

Also, it seems that you only provide runtimes for cifar and no runtimes for the other datasets

**Comment by SSonnyboy** [comment]
I'm sorry it seems to be my problem
